tests/monotouch-test/monotouch-test.csproj

```diff
@@ -422,285 +187,18 @@
-    <Compile Include="CoreData\ManagedObjectModelTest.cs" />
-    <Compile Include="Security\RecordTest.cs" />
-    <Compile Include="Security\SecSharedCredentialTest.cs" />
-    <Compile Include="UIKit\LocalNotificationTest.cs" />
-    <Compile Include="MediaPlayer\MediaItemTest.cs" />
-    <Compile Include="EventKit\RecurrenceRule.cs" />
-    <Compile Include="Foundation\TimerTest.cs" />
-    <Compile Include="AVFoundation\PlayerItemTest.cs" />
-    <Compile Include="UIKit\PanGestureRecognizerTest.cs" />
-    <Compile Include="SystemConfiguration\StatusCodeErrorTest.cs" />
-    <Compile Include="UIKit\CollectionViewControllerTest.cs" />
-    <Compile Include="UIKit\CollectionViewTransitionLayoutTest.cs" />
-    <Compile Include="CoreData\FetchRequestTest.cs" />
-    <Compile Include="MediaPlayer\RemoteCommandCenterTest.cs" />
-    <Compile Include="MediaPlayer\PlayableContentManagerTest.cs" />
-    <Compile Include="MediaPlayer\SkipIntervalCommandTest.cs" />
-    <Compile Include="Foundation\CoderTest.cs" />
-    <Compile Include="Foundation\DecimalNumberTest.cs" />
-    <Compile Include="UIKit\TableViewControllerTest.cs" />
-    <Compile Include="mono\bug18634.cs" />
-    <Compile Include="Foundation\IndexPathTest.cs" />
-    <Compile Include="CoreGraphics\PDFContextTest.cs" />
-    <Compile Include="AudioToolbox\MusicSequenceTest.cs" />
-    <Compile Include="CoreGraphics\GeometryTest.cs" />
-    <Compile Include="CoreGraphics\AffineTransformTest.cs" />
-    <Compile Include="CoreMedia\EnumTest.cs" />
-    <Compile Include="CoreMedia\BlockBufferTest.cs" />
-    <Compile Include="CoreVideo\CoreVideoEnumsTest.cs" />
-    <Compile Include="CoreServices\HttpMessageTest.cs" />
-    <Compile Include="Security\SecureTransportTest.cs" />
-    <Compile Include="UIKit\AppearanceTest.cs" />
-    <Compile Include="CoreGraphics\PDFContentStreamTest.cs" />
-    <Compile Include="CoreGraphics\PDFOperatorTableTest.cs" />
-    <Compile Include="CoreGraphics\PDFScannerTest.cs" />
-    <Compile Include="CoreGraphics\PDFObjectTest.cs" />
-    <Compile Include="Photos\FetchResultTest.cs" />
-    <Compile Include="Foundation\DecimalTest.cs" />
-    <Compile Include="Foundation\NSCharacterSetTest.cs" />
-    <Compile Include="Foundation\FormatterTests.cs" />
-    <Compile Include="CoreImage\CIKernelTests.cs" />
-    <Compile Include="AVFoundation\UtilitiesTest.cs" />
-    <Compile Include="MobileCoreServices\UTTypeTest.cs" />
-    <Compile Include="AVFoundation\SpeechSynthesisVoiceTest.cs" />
-    <Compile Include="CoreBluetooth\PeerTest.cs" />
-    <Compile Include="PassKit\ObjectTest.cs" />
-    <Compile Include="Foundation\EncodingDetectionOptionsTest.cs" />
-    <Compile Include="Foundation\FileCoordinatorTest.cs" />
-    <Compile Include="Foundation\DictionaryContainerTest.cs" />
-    <Compile Include="ObjCRuntime\Messaging.cs" />
-    <Compile Include="ObjCRuntime\TrampolineTest.cs" />
-    <Compile Include="ObjCRuntime\TrampolineTest.generated.cs" />
```

Comment:
Don't remove `TrampolineTest.generated.cs` (and `RegistrarTest.generated.cs`): these files are generated if needed and if they don't exist, but when using a wildcard they don't exist when the wildcard is evaluated, which means they won't be generated because they're not needed since they're not in the list of files to compile.

If you look at the number of monotouch-test tests expected in this PR [1] vs in wrench/master [2], this shows up nicely.

[1] 64-bits iOS Simulator: `Tests run: 1976 Passed: 1939 Inconclusive: 37 Failed: 0 Ignored: 38`
[2] sim64 succeeded: `Tests run: 2188 Passed: 2151 Inconclusive: 37 Failed: 0 Ignored: 38` [wrench](https://wrench.internalx.com/Wrench/ViewLane.aspx?lane_id=1381&host_id=379&revision_id=917553#)

Reply:
those must have already been on my machine when I tested the wildcard version, since they were included when I ran it (and the files were there in VSfM when I opened it).  I actually did check the number of tests run locally before I pushed it.  I'll go add them separately